server/channels/app/admin.go

```diff
@@ -70,15 +70,19 @@ func (s *Server) QueryLogs(rctx request.CTX, page, perPage int, logFilter *model
 		}
 	}
 
+	var appErr *model.AppError
 	serverNames := logFilter.ServerNames
 	if len(serverNames) > 0 {
 		for _, nodeName := range serverNames {
 			if nodeName == "default" {
-				AddLocalLogs(rctx, logData, s, page, perPage, nodeName, logFilter)
+				appErr = AddLocalLogs(rctx, logData, s, page, perPage, nodeName, logFilter)
 			}
 		}
 	} else {
-		AddLocalLogs(rctx, logData, s, page, perPage, serverName, logFilter)
+		appErr = AddLocalLogs(rctx, logData, s, page, perPage, serverName, logFilter)
+	}
+	if appErr != nil {
+		return nil, appErr
```

Comment:
This is slightly changing the behavior, I don't see any problem with the new behavior, but I want to be explicit here about it to be sure that we are considering the implications of this change.

Reply:
I think this is a desired behaviour change. Failing silently leads to missing log messages, which is not obvious to an admin.